x-pack/filebeat/docs/inputs/input-azure-blob-storage.asciidoc

```diff
@@ -130,16 +113,15 @@ processing a certain number of files , decided by the `maximum available workers
 }
 ----
 
-As we can see from the response above, the `message` field contains the original stringified data while the `azure.blob.object.data` contains the objectified data. 
+As we can see from the response above, the `message` field contains the original stringified data. 
     
 *Some of the key attributes are as follows :-* 
 
     1. *message* : Original stringified blob data.
     2. *log.file.path* : Path of the blob in azure cloud.
     3. *azure.blob.container.name* : Name of the container from which the file has been read.
     4. *azure.blob.object.name* : Name of the file/blob which has been read.
-    5. *azure.blob.object.content_type* : Content type of the file/blob. You can find the supported content types <<supported-types,here>> .
-    6. *azure.blob.object.data* :  Objectified blob data , representing the contents of the file.
+    5. *azure.blob.object.content_type* : Content type of the file/blob. You can find the supported content types <<supported-types,here>>.
```

Comment:
`azure.storage.blob.name` and `azure.storage.blob.content_type`?

Reply:
Oops, missed this while updating the docs. Thanks for pointing it out.